### PR TITLE
Add beta tag detection in `<Since>`

### PR DESCRIPTION
**Pull request by FredKSchott**
#### What kind of changes does this PR include?
- Changes to the docs site code

#### Description

Adds a "beta" tag to the `<Since />` component. Without this, it's unclear that a `v2.0.0` feature isn't actually available yet.

#### Previously:

Screen Shot 2023-01-13 at 9 34 01 PM

#### Now:

Screen Shot 2023-01-13 at 9 34 06 PM

### CODE REVIEW

File: src/components/Since.astro
Comment (Yan-Thomas) [inline src/components/Since.astro:29]: Pointing out this `console.log` we probably don't want to be merged.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! All instances of `<Since />` are working properly, only pointed out a `console.log` we probably want removed. Though, I think it's worth to wait for Chris' review before merging.

**Review by delucis — APPROVED** [review]
Nice improvement! Removed the `console.log` pointed out by @Yan-Thomas but LGTM 🙌

**Pull request merged**
